Sources/TuistSupport/MachineEnvironment.swift

```diff
@@ -15,15 +15,29 @@ public class MachineEnvironment: MachineEnvironmentRetrieving {
     private init() {}
 
     /// `clientId` is a unique anonymous hash that identifies the machine running Tuist
-    public let clientId = (Host.current().name?.checksum(algorithm: .md5)) ?? "unknown"
+    public let clientId = (
```

Comment:
If we can't calculate the ID I'd abort. Otherwise, we'll end up with unknowns in the database and it'll take us a while to realize that some logic is not working as expected.
On another note, it's a bit unfortunate that the command that supports an option for normalizing the data into JSON.